api/app/src/providers/oauth2.ts

```diff
@@ -98,6 +95,7 @@ export class OAuth2DefaultImpl implements OAuth2 {
 
     if (this.providerName === "google") {
       uriParams.access_type = "offline";
+      uriParams.prompt = "consent";
     }
```

Comment:
By adding this we will prompt the user to grant access to permissions every time they authenticate with google.

This means if a user authenticates with multiple accounts they will have to grant permssions to their google account every time. Because they grant permissions we will receive a refresh token every time.

Reply:
🎉 🎉 🎉